examples/example2/style.css

```diff
@@ -0,0 +1,128 @@
+.wrapper {
+  padding: 10px;
+}
+
+.form {
+  max-width: 500px;
+  margin: 0 auto;
+  padding: 20px;
+  border-radius: 10px;
+  border: 1px solid rgba(0, 0, 0, 0.1);
+  background-color: #fff;
+}
+
+.form-fields {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-gap: 20px;
+}
+
+.form-group {
+  position: relative;
+}
+
+/* Position the label over the input field (as if its a placeholder) */
+.label {
+  position: absolute;
+  top: 16px;
+  left: 17px;
+  color: #888;
+
+  transition: all 0.05s ease;
+}
+
+/* Move the label to the top on focus or when the field isdirty */
+.form-group.has-focus .label,
+.form-group.is-dirty .label {
```

Comment:
Might be easier to `transform: scale(...)` and use the `transform-origin` to position the label properly. That way its hardware accelerated.

Reply:
Fair point. Will look into that, or if you want to you can change this on monday when I'm not in